doc/build/orm/declarative_mixins.rst

```diff
@@ -273,12 +282,12 @@ or alternatively, the string form (which ultimately generates a lambda)::
     class RefTargetMixin:
         @declared_attr
         def target_id(cls):
-            return Column('target_id', ForeignKey('target.id'))
+            return Column("target_id", ForeignKey("target.id"))
 
         @declared_attr
         def target(cls):
-            return relationship("Target",
-                primaryjoin="Target.id==%s.target_id" % cls.__name__
+            return relationship(
+                Target, primaryjoin=lambda: Target.id == cls.target_id
```

Comment:
ok

Reply:
I think this is "better" since we have access to current class, it's also a bit easier to read. I don't think i need `lambda` here though since it's a `declared_attr`?

Reply:
Yes, I think you can avoid it in this case, but it's fine also with the lambda.

I left a comment here, since it was the only place I found with some changes to the code, other than formatting

Reply:
I also used `f-strings` in some places instead of `%` formatting, since `.format` and `f-strings` are the recommended way as of now

Reply:
OK but this is wrong in context, this doc example is explicitly illustrating string config, the lambda is in the previous example:

```
    The condition above is resolved using a lambda::
            # example with lambda

     or alternatively, the string form (which ultimately generates a lambda)::

          # the changed example is here, and should use as string (or we'd not have this whole paragraph)

```

so this has to change back (or be removed, change it back seems more appropriate at the moment)